tools/scripts/data.toml

```diff
@@ -148,40 +69,3 @@ args = [
     "-v",
 ]
 
-[tasks.hello-world-lang-de]
```

Comment:
Issue: You deleted these tasks with no replacement

Reply:
They're not run on CI and I figured those files will never need updates anymore because we're stable now.

Reply:
I don't like having binary files in the repo with no way to recreate them. They may need to be re-generated in 2.0 or in other similar situations.

Reply:
Do you want me to restore the `--hello-world` functionality on the CLI, or write a script for these?

Reply:
Can you make it work with `--keys core/helloworld@1` ?

Reply:
Yes